src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3092,6 +3092,9 @@ public static void createCeSrRstPinsToVCC(Design design) {
         for (Cell cell : design.getCells()) {
             if (isUnisimFlipFlopType(cell.getType())) {
                 SiteInst si = cell.getSiteInst();
+                if (!Utils.isSLICE(si)) {
+                    continue;
+                }
                 BEL bel = cell.getBEL();
                 Pair<String, String> sitePinNames = pinMapping.get(bel.getBELType());
                 String[] pins = new String[] {"CE", "SR"};
```

Comment:
Not part of this PR, but should we move this `new String[]` to above, outside the loop?

Reply:
Might as well do it here!